pkg/secrets/secrets.go

```diff
@@ -59,6 +59,7 @@ func NewCloudExpander(
 			ctx,
 			slogLogger,
 			cloudConfig.GCPProjectID,
+			"",
```

Comment:
if "" the code use the previous path, I'll defer to the connect team to evaluate if this actually needs to be used in the operator.